public/resources/core/js/controllers/billing-account.controller.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @author Francisco de la Vega <fdelavega@conwet.com>
+ *         Jaime Pajuelo <jpajuelo@conwet.com>
+ *         Aitor Magán <amagan@conwet.com>
+ */
+
+
+(function () {
+
+    'use strict';
+
+    angular
+        .module('app')
+        .controller('BillingAccountSearchCtrl', BillingAccountSearchController)
+        .controller('BillingAccountCreateCtrl', BillingAccountCreateController);
+
+    function BillingAccountSearchController(DATA_STATUS, Utils, BillingAccount) {
+        /* jshint validthis: true */
+        var vm = this;
+
+        vm.list = [];
+        vm.status = DATA_STATUS.LOADING;
+
+        BillingAccount.search().then(function (billingAccounts) {
+            vm.list = billingAccounts;
+            vm.status = DATA_STATUS.LOADED;
+        }, function (response) {
+            vm.errorMessage = Utils.parseError(response, 'Unexpected error trying to retrieve the list of billingAccounts.');
```

Comment:
You are using Utils but it has not been imported and that's the reason why messages are not shown when creating/updating shipping address.